File: tests/fixtures/rename-concept-go/host/transition.go
package renamefixture

import "example.com/renamefixture/legacy"

type Envelope struct {
	LegacyStatus string
}

func Convert(value legacy.LegacyStatus) CanonicalStatus {
	legacyStatus := value
	_ = Envelope{LegacyStatus: string(legacyStatus)}
	return CanonicalStatus(legacyStatus)
}

const retiredCopy = "legacy status"

File: tests/fixtures/php-semantic-family/host/src/Rename/Transition.php
<?php

declare(strict_types=1);

namespace Acme\Rename;

final class Transition
{
    public function convert(LegacyStatus $value): CanonicalStatus
    {
        return new CanonicalStatus();
    }

    public function retiredCopy(): string
    {
        return 'legacy status';
    }
}
